### PR TITLE
Add containerservice managed clusters resource

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

**Review by fdistorted — APPROVED** [review]
Looks good
It also might be useful to add relations to real resources ids

File: resources/container_managed_clusters.go [hunk 1]
```diff
@@ -0,0 +1,973 @@
...
+		Relations: []*schema.Table{
+			{
+				Name:        "azure_container_managed_cluster_pip_user_assigned_identity_exceptions",
+				Description: "ManagedClusterPodIdentityException",
+				Resolver:    fetchContainerManagedClusterPipUserAssignedIdentityExceptions,
+				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"managed_cluster_cq_id", "name"}},
+				Columns: []schema.Column{
+					{
+						Name:        "managed_cluster_cq_id",
+						Description: "Unique ID of azure_container_managed_clusters table (FK)",
```
Comment (roneli) [inline resources/container_managed_clusters.go:442]: Unique CloudQuery ID

File: resources/container_managed_clusters.go [hunk 2]
```diff
@@ -0,0 +1,973 @@
...
+			},
+			{
+				Name:        "azure_container_managed_cluster_private_link_resources",
+				Description: "PrivateLinkResource a private link resource",
+				Resolver:    fetchContainerManagedClusterPrivateLinkResources,
+				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"managed_cluster_cq_id", "id"}},
+				Columns: []schema.Column{
+					{
+						Name:        "managed_cluster_cq_id",
+						Description: "Unique ID of azure_container_managed_clusters table (FK)",
```
Comment (roneli) [inline resources/container_managed_clusters.go:471]: Unique CloudQuery ID

File: resources/container_managed_clusters.go [hunk 3]
```diff
@@ -0,0 +1,973 @@
...
+				},
+			},
+			{
+				Name:        "azure_container_managed_cluster_agent_pool_profiles",
+				Description: "ManagedClusterAgentPoolProfile profile for the container service agent pool",
+				Resolver:    fetchContainerManagedClusterAgentPoolProfiles,
+				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"managed_cluster_cq_id", "name"}},
+				Columns: []schema.Column{
+					{
+						Name:        "managed_cluster_cq_id",
```
Comment (roneli) [inline resources/container_managed_clusters.go:517]: Unique CloudQuery ID

**Review by roneli — APPROVED** [review]
LGTM, Minor description changes on _cq_id columns, and we can merge

File: resources/container_managed_clusters.go (hunk 1 above)
Comment (irmatov) [reply resources/container_managed_clusters.go:442]: Updated

File: resources/container_managed_clusters.go (hunk 3 above)
Comment (irmatov) [reply resources/container_managed_clusters.go:517]: Updated

File: resources/container_managed_clusters.go (hunk 2 above)
Comment (irmatov) [reply resources/container_managed_clusters.go:471]: Updated

**Pull request merged**